Yoast/YoastSEO.js
# Remove the rounded border from the snippet preview

## Summary

Remove the very light rounded border from the snippet preview, since it affects the snippet preview width. See https://github.com/Yoast/wordpress-seo/issues/6374

## Test instructions

- check the browserified example
- in your browser's dev tools, make the `.form-container` larger so the snippet preview gets wider till its max-width
- check the snippet preview width is actually 598px, it should be 600px (see screenshot below)

![screen shot 2017-01-13 at 13 23 29](https://cloud.githubusercontent.com/assets/1682452/21930175/dd436352-d994-11e6-8a4b-f660365936ab.png)

- switch to this branch and check the border is gone and the width highlighted in the screenshot above is now `600px`

Fixes #1030
